docs/rst/installation/linux/amldashboard.rst

```diff
@@ -0,0 +1,71 @@
+.. include:: /rst/exports/alias.include
+.. include:: /rst/exports/roles.include
+
+.. _amldashboard_linux:
+
+#############
+AML Dashboard
+#############
+
+To get started with *AML Dashboard*, follow these steps.
+
+Clone the repository using the following command:
+
+.. code-block:: bash
+
+    git clone https://github.com/eProsima-Private/AML-Dashboard.git
+
+Requirements
+------------
+
+If |amlip| is not already installed in your system, follow the installation instructions provided `here <https://aml-ip.readthedocs.io/en/latest/rst/installation/linux.html>`__.
+
+Python dependencies
+^^^^^^^^^^^^^^^^^^^
+
+Ensure you have **python3.11** installed together with the packages listed in the ``requirements.txt`` file. If not, install them using the following commands: 
```

Comment:
Could you please add a link to the `requirements.txt` file.

Reply:
Done